backend/accounts.sql

```diff
@@ -0,0 +1,13 @@
+-- @block 
+CREATE TABLE IF NOT EXISTS account (
+    accountID INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    email VARCHAR(255) NOT NULL,
+    encryptedPass VARCHAR(255) NOT NULL,
+    displayTheme int NOT NULl,
+    phoneNum VARCHAR(20),
```

Comment:
It seems that the longest is 15 including country code, so I will probably keep it at 15. I kept it null in case we wanted to allow email as the sole identifier.